### DISCUSSION TITLE
Unable to Push commits

### DISCUSSION THREAD

**Discussion by BikramSankhari** [community/community#127992] [New to GitHub]
I am using Windows OS and followed the github docs for SSH Authentication.

Now when I am writing this line of code

```
ssh -T git@github.com
```

I mam getting this output

```
Hi Bikram-Sankhari! You've successfully authenticated, but GitHub does not provide shell access.
```

But when I am trying to push to GitHub I am getting this error
```
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Can anyone help me out please??

**Comment by DaviZCodes** [comment]
This might resolve your issue: https://stackoverflow.com/questions/12940626/github-error-message-permission-denied-publickey
